docs/src/contribute/linter/index.md

```diff
@@ -63,7 +55,6 @@ These docs follow [Divio's Documentation System](https://docs.divio.com/document
     - [`rustc_middle`](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/index.html)
         - ⭐️ [`TyCtxt`](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/ty/context/struct.TyCtxt.html)
         - ⭐️ [`Ty`](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/ty/struct.Ty.html) (Not to be confused with HIR `Ty`)
-        - [`Map`](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/hir/map/struct.Map.html) (HIR)
```

Comment:
`hir::Map` has been removed, as all of its capabilities have been moved to `TyCtxt`. As per <https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/hir/map/index.html>:

> This module used to contain a type called `Map`. That type has since been eliminated, and all its methods are now on `TyCtxt`. But the module name stays as `map` because there isn't an obviously better name for it.